.gitattributes

```
@@ -0,0 +1,2 @@
+/.editorconfig     export-ignore
```

```suggestion
.editorconfig     export-ignore
```

du coup pour les sous .editorconfig